### PR TITLE
Added documentation to ScalePictureToFit + allow disabling the property programmatically.

### PR DESCRIPTION
**Pull request by Momoumar**
Added documentation to ```ScalePictureToFit``` method and allow disabling the property programmatically.

Ref issue #518

Link to companion for testing: https://drive.google.com/file/d/0Bw0fmqX0oC9YaXZYQXRKZTVmb0k/view?usp=sharing

### CODE REVIEW

File: appinventor/components/src/com/google/appinventor/components/runtime/Image.java [hunk 1]
```diff
@@ -107,10 +107,12 @@ public void Picture(String path) {
 
   @DesignerProperty(editorType = PropertyTypeConstants.PROPERTY_TYPE_BOOLEAN,
       defaultValue = "False")
-  @SimpleProperty
+  @SimpleProperty(description = "Specifies whether the image should be resized to match the size                                 of the ImageView.")
```
Comment (josmas) [inline appinventor/components/src/com/google/appinventor/components/runtime/Image.java]: is there an issue with the indentation here or is it github showing things *funnily*?

Comment (Momoumar) [inline appinventor/components/src/com/google/appinventor/components/runtime/Image.java]: Right. The formatting was not correct.
Thanks @josmas

**Comment by halatmit** [comment]
LGTM

**Comment by jisqyv** [comment]
@Momoumar @halatmit Moving this to gerrit for the UCR branch

**Comment by jisqyv** [comment]
MERGED INTO "ucr"

**Pull request closed without merging**